spec/index.bs

```diff
@@ -1324,7 +1330,110 @@ dictionary GPUBufferBinding {
 };
 </script>
 
-  * {{GPUBufferBinding/size}}: If undefined, specifies the range starting at {{GPUBufferBinding/offset}} and ending at the end of the buffer.
+  * {{GPUBufferBinding/size}}: If undefined, specifies the range starting at
+      {{GPUBufferBinding/offset}} and ending at the end of the buffer.
+
+A {{GPUBindGroup}} object has the following internal slots:
+
+<dl dfn-type=attribute dfn-for="GPUBindGroup">
+    : <dfn>\[[layout]]</dfn> of type {{GPUBindGroupLayout}}.
+    ::
+        The {{GPUBindGroupLayout}} this {{GPUBindGroup}} associates.
+
+    : <dfn>\[[bindings]]</dfn> of type sequence<{{GPUBindGroupBinding}}>.
+    ::
+        The set of {{GPUBindGroupBinding}}s this {{GPUBindGroup}} describes.
+</dl>
+
+### {{GPUDevice/createBindGroup()|GPUDevice.createBindGroup(GPUBindGroupDescriptor)}} ### {#GPUDevice-createBindGroup}
+
+<div algorithm="GPUDevice.createBindGroup(descriptor)">
+
+The <dfn method for="GPUDevice">createBindGroup(|descriptor|)</dfn> method is used to create
+    {{GPUBindGroup}}s.
+
+1. Ensure [=bind group device validation=] is not violated.
+1. Let |bindGroup| be a new valid {{GPUBindGroup}} object.
+1. Ensure {{GPUBindGroupLayout}} in |descriptor|.{{GPUBindGroupDescriptor/layout}} is valid.
+1. Ensure the number of {{GPUBindGroupLayoutDescriptor/bindings}} of
+    |descriptor|.{{GPUBindGroupDescriptor/layout}} exactly equals to the number of
+    |descriptor|.{{GPUBindGroupDescriptor/bindings}}.
```

Comment:
would become unnecessary with [1]

Reply:
Per my understanding, the suggestion [1] will ensure GPUBindGroupLayout.bindings' number >= GPUBindGroup.bindings' number, but not vice versa. So we still need this rule. But please correct me if I made a mistake here.

Reply:
Ah yeah, you're right.